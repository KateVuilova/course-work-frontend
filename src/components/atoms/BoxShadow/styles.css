.BoxShadow {
  --BoxShadow-shadow: 0 2px 4px 0 rgba(136, 158, 176, .4);

  box-shadow: var(--BoxShadow-shadow);
  background-color: white;
  border-radius: var(--runic-border-radius);
  height: 500px;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 50px;
}
