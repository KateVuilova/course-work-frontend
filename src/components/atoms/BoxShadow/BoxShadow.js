import React, { Component } from 'react';
import classNames from 'classnames';
import './styles.css';

export default class BoxShadow extends Component {
  getClassName = (className) =>
    classNames('BoxShadow', className)

  render () {
    return (
      <div className={this.getClassName(this.props.className)}>
        {this.props.children}
      </div>
    );
  }
}
