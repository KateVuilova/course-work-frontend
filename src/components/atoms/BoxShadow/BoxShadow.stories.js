import React from 'react';
import { storiesOf } from '@storybook/react';
import BoxShadow from './BoxShadow';

storiesOf('Atoms / BoxShadow', module).add('Default', () => (
  <BoxShadow>
    <div style={{ height: '100px', padding: '25px' }}>Hello!</div>
  </BoxShadow>
));
