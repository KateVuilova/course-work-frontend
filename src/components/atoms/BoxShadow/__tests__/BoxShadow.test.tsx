import React from 'react';
import { shallow } from 'enzyme';
import BoxShadow from '../BoxShadow';

describe('BoxShadow', () => {
  it('should render', () => {
    const wrapper = shallow(<BoxShadow />);
    expect(wrapper).toMatchSnapshot();
  });
});
