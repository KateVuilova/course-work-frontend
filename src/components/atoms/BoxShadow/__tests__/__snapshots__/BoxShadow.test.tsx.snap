// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxShadow should render 1`] = `
<div
  className="BoxShadow"
/>
`;
